src/sections/collection/datasets-list/NoDatasetsMessage.tsx

```diff
@@ -14,7 +14,7 @@ export function NoDatasetsMessage() {
       ) : (
         <Trans i18nKey="noDatasetsMessage.anonymous">
           <p>
-            This dataverse currently has no datasets. Please <a href={Route.LOG_IN}>log in</a> to
+            This collection currently has no datasets. Please <a href={Route.LOG_IN}>log in</a> to
             see if you are able to add to it.
           </p>
```

Comment:
There is a locale for this already, its the `i18nKey` that the Trans component is using correctly.
The text inside open and closing tags of Trans that you see is a fallback that you can add in case the Trans component can't find the locale.